community/event-funding.tt

```diff
@@ -0,0 +1,105 @@
+[% WRAPPER layout.tt title="Event funding" handlesLayout=1 %]
+
+<div class="page-title">
+  <div>
+    <a href="[% root%]community/index.html">Community</a>
+    <span>→</span>
+  </div>
+  <h1>Event funding</h1>
+</div>
+
+<section class="community-event-funding">
+  <p>
+    The NixOS community is awesome! You guys have been generously donating
+    throughout the year to keep the NixOS projects running smoothly. We want to
+    show our appreciation by launching a new grant program that will help our
+    community thrive even more.
+  </p>
+  <p>
+    This program will be aimed at supporting and boosting the NixOS community,
+    which has been instrumental in getting us where we are today. We hope that
+    it will continue to reward and encourage amazing folks like you to
+    volunteer your time and talent for years to come.
+  </p>
+
+  <div class="community-event-funding-button">
+    <a href="https://commotion.page/view/event-funding">Apply</a>
+  </div>
+
+  <h2>Who can apply?</h2>
+  <ul>
+    <li>
+      Any active community member that’s looking to organize an event with Nix
+      developers, publicly announced.
+    </li>
+    <li>
+      Recipient must be in a SEPA country or in a country supported by Wise
+      (India excluded as it proved to be difficult unless you have a company
+      there)
+    </li>
+  </ul>
+
+  <h2>Examples of things that this is intended for</h2>
+  <ul>
+    <li>
+      Nix meetups, where a few people can make a presentation to showcase Nix
+      to the audience
+    </li>
+    <li>
+      Sprints, focused hacking for a period of time
+    </li>
+    <li>
+      Smaller Nix based togethers
+    </li>
+  </ul>
```

Comment:
ADD: If you are planning an event that doesn't look like any of these, but you think it may be important for the Nix community, feel free to reach out. We're not rigid!